src/olympia/api/urls.py

```diff
@@ -29,6 +29,7 @@
     url(r'^', include('olympia.signing.urls')),
     url(r'^', include(amo_api_patterns)),
     url(r'^hero/', include('olympia.hero.urls')),
+    url(r'^scanner/', include('olympia.scanners.api_urls')),
```

Comment:
Do we want this available in /v4/ or just /v5/ ? The test specifies v5 but it's been added to v4 here.  (it's not difficult to change - just define a `v5_api_urls = v4_api_urls + [url(r'^scanner...]`)

Reply:
could be in v4+